contracts/contracts/token/OUSD.sol

```diff
@@ -32,40 +27,62 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
     );
     event AccountRebasingEnabled(address account);
     event AccountRebasingDisabled(address account);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+    event Approval(
+        address indexed owner,
+        address indexed spender,
+        uint256 value
+    );
 
     enum RebaseOptions {
         NotSet,
-        OptOut,
-        OptIn
+        StdNonRebasing,
+        StdRebasing,
+        YieldDelegationSource,
+        YieldDelegationTarget
     }
 
+    // Add slots to align with deployed OUSD contract
+    uint256[154] private _gap;
     uint256 private constant MAX_SUPPLY = ~uint128(0); // (2^128) - 1
```

Comment:
Let's demagic this to just `type(uint128).max`

Reply:
done here: https://github.com/OriginProtocol/origin-dollar/pull/2298/commits/a1d3cdc27789729e3df9fbd6c7d195e2594643d9